src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryTypeSize.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.bigquery.util;
+
+/**
+ * Stores the Precision and Scale for the different Numeric (Decimal) types
+ * provided by BigQuery
+ */
+public final class BigQueryTypeSize {
```

Comment:
I would add a link here to the BQ docs where this specification is outlined, just for future reference

Reply:
fixed